zipkin-common/src/main/scala/com/twitter/zipkin/common/Span.scala

```diff
@@ -20,25 +20,35 @@ import com.twitter.zipkin.Constants
 import com.twitter.zipkin.util.Util._
 
 /**
- * A span represents one RPC request. A trace is made up of many spans.
+ * A trace is a series of spans (often RPC calls) which form a latency tree.
  *
- * A span can contain multiple annotations, some are always included such as
- * Client send -> Server received -> Server send -> Client receive.
+ * <p/>Spans are usually created by instrumentation in RPC clients or servers, but can also
+ * represent in-process activity. Annotations in spans are similar to log statements, and are
+ * sometimes created directly by application developers to indicate events of interest, such as a
+ * cache miss.
  *
- * Some are created by users, describing application specific information,
- * such as cache hits/misses.
+ * <p/>The root span is where [[traceId]] = [[id]] and [[parentId]] is empty. The root span is
```

Comment:
If you want to change this, we should log an issue because all core instrumentation currently set  traceid = spanid for root spans (ex finagle, ruby tracer, brave, as well zipkin has logic around this)

Basically, I'm not opposed to changing this, but it is in fact a change, and likely has impact. I don't want to document something that isn't status quo and also breaks things.